src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php

```diff
@@ -334,169 +262,73 @@ protected function checkMasterBranch($hookData, Github $github, Logger $logger,
 	/**
 	 * Checks for a PR-<branch> label
 	 *
-	 * @param   object  $hookData  Hook data payload
-	 * @param   Github  $github    Github object
-	 * @param   Logger  $logger    Logger object
-	 * @param   object  $project   Object containing project data
+	 * @param   object       $hookData  Hook data payload
+	 * @param   Github       $github    Github object
+	 * @param   Logger       $logger    Logger object
+	 * @param   object       $project   Object containing project data
+	 * @param   IssuesTable  $table     Table object
 	 *
 	 * @return  void
 	 *
 	 * @since   1.0
 	 */
-	protected function checkPullLabel($hookData, Github $github, Logger $logger, $project)
+	protected function checkPullLabels($hookData, Github $github, Logger $logger, $project, IssuesTable $table)
 	{
 		// Set some data
-		$issueLabel       = 'PR-' . $hookData->pull_request->base->ref;
+		$prLabel          = 'PR-' . $hookData->pull_request->base->ref;
 		$languageLabel    = 'Language Change';
 		$addLabels        = array();
-		$prLabelSet       = false;
-		$languageLabelSet = false;
+		$removeLabels     = array();
+		$prLabelSet       = $this->checkLabel($hookData, Github $github, Logger $logger, $project, IssuesTable $table, $prLabel);
```

Comment:
You should remove type hinting here. And also do not forget to remove $table as it is not used.